### PR TITLE
SDK updates to work with latest Android SDK build and support tools

### PR DESCRIPTION
**Pull request by shdunning**

### CODE REVIEW

File: UVDemo/UVDemo.iml [hunk 1]
```diff
@@ -27,72 +24,105 @@
...
-    <orderEntry type="library" exported="" name="support-v4-21.0.3" level="project" />
-    <orderEntry type="library" exported="" name="support-annotations-21.0.3" level="project" />
+    <orderEntry type="library" exported="" name="appcompat-v7-24.2.1" level="project" />
+    <orderEntry type="library" exported="" name="support-vector-drawable-24.2.1" level="project" />
     <orderEntry type="library" exported="" name="okhttp-2.4.0" level="project" />
+    <orderEntry type="library" exported="" name="signpost-core-1.2.1.2" level="project" />
     <orderEntry type="library" exported="" name="okio-1.4.0" level="project" />
-    <orderEntry type="module" module-name="UserVoiceSDK" exported="" />
   </component>
 </module>
```
Comment (thuytrinh) [inline UVDemo/UVDemo.iml]: Should u ignore these `*.iml` files too?

File: UVDemo/build.gradle [hunk 2]
```diff
@@ -67,6 +67,6 @@ tasks.whenTaskAdded { theTask ->
 }
 
 dependencies {
-    compile 'com.android.support:support-v4:20.+'
+    compile 'com.android.support:support-core-utils:24.2.1'
```
Comment (thuytrinh) [inline UVDemo/build.gradle]: 👍

File: UserVoiceSDK/build.gradle [hunk 3]
```diff
@@ -50,8 +50,9 @@ android {
 }
 
 dependencies {
-    compile 'com.android.support:support-v4:21.'
-    compile 'com.android.support:appcompat-v7:21.+'
+    // compile 'com.android.support:support-v4:21.+'
+    compile 'com.android.support:support-core-utils:24.2.1'
+    compile 'com.android.support:appcompat-v7:24.2.1'
```
Comment (thuytrinh) [inline UserVoiceSDK/build.gradle]: 👍

File: UVDemo/UVDemo.iml (hunk 1 above)
Comment (shdunning) [inline UVDemo/UVDemo.iml]: Yes, this should not be under source control -- good catch!

**Pull request merged**